hack/feature-wait.sh

```diff
@@ -12,15 +12,20 @@ if [ "$FEATURES" == "" ]; then
 	exit 1
 fi
 
-ATTEMPTS=0
-MAX_ATTEMPTS=200
 all_ready=false
 export TEST_SUITES="validationsuite"
 export FAIL_FAST="-ginkgo.failFast"
 export DONT_REBUILD_TEST_BINS=true
+export TIMEOUT="${TIMEOUT:-5400}" # results in 90 minutes timeout
```

Comment:
please just add a small `echo` to tell how much time we are going to wait until timeout